pkg/local_workflows/output_workflow.go

```diff
@@ -79,7 +79,14 @@ func filterSummaryOutput(config configuration.Configuration, input workflow.Data
 	}
 
 	workflowId := workflow.NewTypeIdentifier(WORKFLOWID_OUTPUT_WORKFLOW, "FilterTestSummary")
-	return workflow.NewDataFromInput(input, workflowId, content_type.TEST_SUMMARY, bytes), nil
+	output := workflow.NewDataFromInput(
```

Comment:
Suggestion: Let's limit impact of the changes and not use this function everywhere.